chemprop/multitask_utils.py

```diff
@@ -20,11 +19,11 @@ def reshape_values(
     :param test_data: A :class:`~chemprop.data.MoleculeDataset` containing valid datapoints.
     :param natom_targets: The number of atomic targets.
     :param nbond_targets: The number of bond targets.
-    :param num_tasks: Number of tasks.
     :return: List of atomic/bond properties with shape
              (data_size, num_tasks, number of atomic/bond properties for this data in each task).
     """
     n_atoms, n_bonds = test_data.number_of_atoms, test_data.number_of_bonds
+    num_tasks = natom_targets + nbond_targets
```

Comment:
It seems that "num_task" is used specifically for atom and bond targets, and not for multitask molecular targets prediction. Maybe consider adding a comment or adjust the variable name so that it is not confused with the cases for molecular targets.

Reply:
Thanks for the comment. I have adjusted the variable name.